src/Routes/RoutesCommon.tsx

```diff
@@ -151,6 +153,16 @@ export const routes: NestedRoute[] = [
     category: RouteCategory.Featured,
     disable: !getConfig().REACT_APP_FEATURE_RFOX,
   },
+  {
+    path: '/fox-benefits',
+    label: 'navBar.foxBenefits',
+    icon: <FoxIcon />,
+    main: FoxPage,
```

Comment:
q: do we want to consolidate terminology here i.e either FOX Page or FOX benefits? Seeing the route is `fox-benefits`, think `fox` as a route may be better and more generic since we don't know whether or not we may change the page name

Reply:
Agree, changed in [e05e4b2](https://github.com/shapeshift/web/pull/7882/commits/e05e4b244591c3a87f0da0fca531273c9a08a4c3)